### PR TITLE
Remove unnecessary config variables, add objects to __init__

### PR DESCRIPTION
**Pull request by alexfurmenkov**

### CODE REVIEW

File: cdisc_rules_engine/config/config.py [hunk 1]
```diff
@@ -20,10 +20,6 @@ def __new__(cls):
                 "REDIS_HOST_NAME",
                 "REDIS_ACCESS_KEY",
                 "CDISC_LIBRARY_API_KEY",
-                "CDISC_LIBRARY_URL",
-                "DICTIONARIES_TABLE_NAME",
-                "WHODRUG_TERMS_TABLE_NAME",
-                "MEDDRA_TERMS_TABLE_NAME",
                 "DATA_SERVICE_TYPE",
```
Comment (alexfurmenkov) [inline cdisc_rules_engine/config/config.py:23]: CDISC_LIBRARY_URL got changed to a hardcoded value, table names should not be set here

**Comment by nhaydel** [comment]
Looks good. Aren't we merging into the development branch though?

File: cdisc_rules_engine/config/config.py (hunk 1 above)
Comment (DmitryMK) [reply cdisc_rules_engine/config/config.py:23]: Does it makes sense to keep CDISC Library API customizable? In Parexel we work with the CDISC Library via a mirror.

Comment (DmitryMK) [reply cdisc_rules_engine/config/config.py:23]: It will be a separate PR, so it is more like a question whether we should do it later or not.

Comment (alexfurmenkov) [reply cdisc_rules_engine/config/config.py:23]: Yes, we can implement it later in a separate PR

**Comment by alexfurmenkov** [comment]
> Looks good. Aren't we merging into the development branch though?

Will change the target, thanks

**Comment by alexfurmenkov** [comment]
@nhaydel changed

**Review by nhaydel — APPROVED** [review]

**Pull request merged**